l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -53,6 +53,8 @@ def saveAttachment(self, fatturapa, number):
 
         attach_str = fatturapa.to_xml(self.env)
         attach_vals = {
+            "res_model": "fatturapa.attachment.out",
+            "res_id": -1,
```

Comment:
Grazie della PR!
Così com'è non la vedo come una modifica facilmente mantenibile: ad esempio per sapere se è necessaria nella altre versioni (comprese migrazioni future), bisogna provare a mano i passi che hai descritto nella issue.

Per renderla più mantenibile, puoi aggiungere un test?

Una modifica meno _hack_ potrebbe essere fare override del metodo `ir.attachment.check` per gestire il caso in cui l'allegato è collegato a una fattura elettronica.

Potresti chiarire come mai `res_id` deve essere proprio -1?
Normalmente è l'ID del record cui è collegato l'allegato.

Reply:
> Una modifica meno _hack_ potrebbe essere fare override del metodo `ir.attachment.check` per gestire il caso in cui l'allegato è collegato a una fattura elettronica.

+1 ma quelle righe sono atomiche nel metodo, faresti un override completo ?

Reply:
> > Una modifica meno _hack_ potrebbe essere fare override del metodo `ir.attachment.check` per gestire il caso in cui l'allegato è collegato a una fattura elettronica.
> 
> +1 ma quelle righe sono atomiche nel metodo, faresti un override completo ?

Cosa sarebbe un _override completo_?

Intendevo una modifica del tipo:
```
if `è un allegato di fattura elettronica`:
    # Fai i controlli necessari in questo caso
else:
    return super()...
```

Reply:
Intendo che non so se è possibile bypassare solo [questo controllo](https://github.com/odoo/odoo/blob/551bf83f792813535cb571b8d746238d5fea6b50/odoo/addons/base/models/ir_attachment.py#L434-L435) chiamando super() , devi andare a modificare il metodo nel suo insieme o sbaglio?

Reply:
> Intendo che non so se è possibile bypassare solo [questo controllo](https://github.com/odoo/odoo/blob/551bf83f792813535cb571b8d746238d5fea6b50/odoo/addons/base/models/ir_attachment.py#L434-L435) chiamando super() , devi andare a modificare il metodo nel suo insieme o sbaglio?

Fare override vuol dire ridefinire un certo **metodo** chiamando `super` per eseguire il codice del **metodo** ridefinito, che io sappia non esistono override che permettono di modificare il codice solo di parte del **metodo**.

Ci si può andare a complicare la vita con il `context`, ma eviterei.

Reply:
@SirAionTech Si mi sono espresso male, stavo cercando di capire se si potesse tentare una soluzione di quel tipo, ma diventa macchinoso correggere il metodo, forse conviene tenere la soluzione attuale

Reply:
> Una modifica meno _hack_ potrebbe essere fare override del metodo `ir.attachment.check` per gestire il caso in cui l'allegato è collegato a una fattura elettronica

Sequendo questa idea ho fatto https://github.com/OCA/l10n-italy/pull/3827